catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -204,4 +261,79 @@ public void setParser(Parser parser) {
         this.marshalConfigurator.addProperty(Marshaller.JAXB_FRAGMENT, true);
         this.parser = parser;
     }
+
+    public SourceMonitor registrySourceMonitor = new SourceMonitor() {
```

Comment:
I don't think so, it also probably shouldn't have been public. Will move/update.